### PR TITLE
feat: add meson

### PR DESCRIPTION
**Pull request by smira**
meson depends on python3 + _ctypes, so I had to add libffi for ctypes.
ctypes build was broken with python 3.9.6, so I updated python to 3.10.2

Signed-off-by: Andrey Smirnov <andrey.smirnov@talos-systems.com>

### CODE REVIEW

File: meson/pkg.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+name: meson
+dependencies:
+  - stage: base
+  - stage: libffi
+  - stage: python3
+steps:
+  - sources:
+      - url: https://github.com/mesonbuild/meson/releases/download/0.61.1/meson-0.61.1.tar.gz
```
Comment (eli-schwartz) [inline meson/pkg.yaml:8]: By a strange coincidence, Meson 0.61.2 was released about 20 minutes before this PR was opened.

Comment (smira) [reply meson/pkg.yaml:8]: thanks for noticing that!

**Review by frezbo — APPROVED** [review]

**Comment by smira** [comment]
updated with s/LibreSSL/OpenSSL/g in the tools

**Review by andrewrynhard — APPROVED** [review]

**Pull request merged**